lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb

```diff
@@ -1,8 +1,9 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
-    <ul id="archives">
+  <h3 class="sidebar-title"><%= sidebar.title %></h3>
+  <div class="sidebar-body">
+    <ul>
       <% sidebar.archives.each do |month| %>
+      <!-- ##ERROR WITH ARTICLE COUNT  -->
```

Comment:
My b. I meant to delete this prior to committing.

This was meant to be used for my own purposes. I'm commenting `##ERROR` so I can find where I left off should I accidentally close a tab.